package.json

```diff
@@ -52,6 +51,7 @@
     "@sendbird/chat": "^4.2.3",
     "css-vars-ponyfill": "^2.3.2",
     "date-fns": "^2.16.1",
+    "lamejs": "^1.2.1",
```

Comment:
To add lame
```
1. Link to LAME as separate jar (lame.min.js or lame.all.js) 

> for this, add lamejs to rollup.externals -> https://github.com/sendbird/sendbird-uikit-react/blob/main/rollup.config.js#L40
https://rollupjs.org/configuration-options/#external

2. Fully acknowledge that you are using LAME, and give a link
   to our web site, lame.sourceforge.net
   
   > Add lameJS to our readme

3. If you make modifications to LAME, you *must* release these
   these modifications back to the LAME project, under the LGPL.
   
> not applicable
```

Reply:
Thanks let me follow-up this in the other PR
[UIKIT-3259](https://sendbird.atlassian.net/browse/UIKIT-3259)